index.html

```diff
@@ -1,348 +1,328 @@
-                        you see and have a project you need coded,
-                        don’t hestiate to contact me.
-                    </p>
                     <div class="green-btn">
-                        <a href="">Get My Resume</a>
+                        <a href="single-project.html">See Project</a>
                     </div>
                 </div>
-                <ul class="grid-container skills-set">
-                    <li class="grid-item skill-item">
-                        <div class="item-icon">
-                            <img src="assets/images/lang-icon.svg" alt="icon" />
+
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
+                    </div>
+                    <div class="item-description">
+                        <div class="item-title">
+                            <h2>Mult-Post Stories</h2>
+                            <h3>Gain + Glory</h3>
                         </div>
-                        <h3 class="item-title">Languages</h3>
-                        <ul class="item-langs">
+                        <ul class="technologies">
                             <li>Javascript</li>
-                            <li>Ruby</li>
+                            <li>Ruby on rails</li>
                             <li>HTML</li>
                             <li>CSS</li>
                         </ul>
-                    </li>
+                    </div>
+                    <div class="green-btn">
+                        <a href="single-project.html">See Project</a>
+                    </div>
+                </div>
 
-                    <li class="grid-item skill-item">
-                        <div class="item-icon">
-                            <img src="assets/images/framework-icon.svg"
-                                alt="icon" />
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
+                    </div>
+                    <div class="item-description">
+                        <div class="item-title">
+                            <h2>Mult-Post Stories</h2>
+                            <h3>Gain + Glory</h3>
                         </div>
-                        <h3 class="item-title">Frameworks</h3>
-                        <ul class="item-langs">
-                            <li>Bootstrap</li>
-                            <li>Ruby on Rails</li>
-                            <li>RSpec</li>
-                            <li>Capybara</li>
-                            <li>Selenium</li>
+                        <ul class="technologies">
+                            <li>Javascript</li>
+                            <li>Ruby on rails</li>
+                            <!-- <li>HTML</li>
+                            <li>CSS</li> -->
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.